Azure/sonic-buildimage
# [Fastboot] Delay PMON service for better fastboot performance

Signed-off-by: Shlomi Bitton <shlomibi@nvidia.com>



#### Why I did it

Profiling the system state on init after fast-reboot during create_switch function execution, it is possible to see few python scripts running at the same time.
This parallel execution consume CPU time and the duration of create_switch is longer than it should be.
Following this finding, and the motivation to ensure these services will not interfere in the future, PMON is delayed in 90 seconds until the system finish the init flow after fastboot.

![bootchart_no_optimizations](https://user-images.githubusercontent.com/60430976/158380853-69fc4e7a-27a3-4caa-84d2-3c463bfb19ab.png)

#### How I did it

- Add a timer for PMON service.
- Exclude for MLNX platform the start trigger of PMON when SYNCD starts in case of fastboot.
- Copy the timer file to the host bin image.

#### How to verify it

Run fast-reboot on MLNX platform and observe faster create_switch execution time.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
